src/v/kafka/server/handlers/describe_groups.cc

```diff
@@ -22,6 +23,16 @@
 
 namespace kafka {
 
+ss::future<described_group> describe_group(
```

Comment:
can this be in an anonymous namespace / made static?

Reply:
done